tools/docker/about.md

```diff
@@ -28,3 +28,9 @@ For instructions how to configure the Docker engine see [configuring Docker](/to
 ## Featured images
 
 Find out about [released images](/tools/docker/docker-images.html).
+
+## Alternatives
+
+- [Podman](https://podman.io/) 
+  - Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System.
```

Comment:
I see we don't have the page. I'll add it. please point to:

```suggestion
### [Podman](/tools/docker/about.html) 
Podman is a daemonless container engine for developing, managing, and running OCI Containers. Usage is mostly same as with docker.

You can install it with the following command:
```console
$ sudo dnf install podman
\```  
```

I've also updated the formatting, which seemed strange to me. But the `\` don't belong there, please remove it manualy.

Reply:
I will remove the command when I add the page (/tools/podman/about.html).